.travis.yml

```diff
@@ -12,7 +12,7 @@ before_install:
 
 install:
   - npm install
-  - npm install cucumber
+  - npm install cucumber@1.0.0
```

Comment:
@peter-reperger does the current code work with more recent versions of cucumber or just version 1.0.0?  The version number is quite specific in the npm install.  I.e. it's not `~1.0.0` or `^1.0.0` (https://docs.npmjs.com/getting-started/semantic-versioning)